provider/schema/resolvers.go

```diff
@@ -38,3 +42,139 @@ func ParentPathResolver(path string) ColumnResolver {
 		return r.Set(c.Name, funk.Get(r.Parent.Item, path, funk.WithAllowZero()))
 	}
 }
+
+// DateResolver resolves the different date formats (ISODate - 2011-10-05T14:48:00.000Z is default) into *time.Time
+//
+// Examples:
+// DateResolver("Date", "") - resolves using RFC.RFC3339 as default
+//
+// DateResolver("InnerStruct.Field", time.RFC822)  - resolves using time.RFC822
+//
+// DateResolver("InnerStruct.Field", "2011-10-05")  - resolves using manually set format
+func DateResolver(path string, rfc string) ColumnResolver {
```

Comment:
I would add support for multiple rfcs with ...string so all rfcs are tried by that order